### PR TITLE
[Communication Services - Check Routing] Added /sip:testRoutesWithNumber endpoint

### LINKED ISSUE #20500: [Communication Services - Check Routing]  REST API Review Meeting
<div data-wrapper="true" style="font-size:9pt;font-family:'Segoe UI','Helvetica Neue',sans-serif;"><div>This meeting was created by Karol Druzycki.</div>
<div>&nbsp;</div>
<div>It will be used to Track the conversation in the informational Session for the Azure Communication Services Service.</div>
<div>&nbsp;</div>
<div>Detailed meeting information and documents provided can be accessed&nbsp;<a href="https://web.powerapps.com/apps/e41c69de-ef96-44ba-b287-0e012e7c13ae?view=edit&event=b7ae24e1-3c29-ed11-9db1-000d3a30e8ee">here</a></div></div>

### PR DESCRIPTION
**Pull request by kdruzycki**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>
Azure 1st Party Service can try out the [Shift Left](https://aka.ms/ShiftLeft) experience to initiate API design review from ADO code repo.  If you are interested, may request engineering support by filling in with the form https://aka.ms/ShiftLeftSupportForm.

### Changelog
Add a changelog entry for this PR by answering the following questions:
  1. What's the purpose of the update?
      - [ ] new service onboarding
      - [x] new API version
      - [ ] update existing version for new feature
      - [ ] update existing version to fix swagger quality issue in s360
      - [ ] Other, please clarify
  2. When are you targeting to deploy the new service/feature to public regions? Please provide the date or, if the date is not yet available, the month.
  3. When do you expect to publish the swagger? Please provide date or, the the date is not yet available, the month.
  4. If updating an existing version, please select the specific language SDKs and CLIs that must be refreshed after the swagger is published.
      - [ ] SDK of .NET (need service team to ensure code readiness)
      - [x] SDK of Python
      - [x] SDK of Ruby
      - [x] SDK of TypeScript
      - [ ] SDK of Java
      - [ ] SDK of Js
      - [ ] SDK of Go
      - [ ] PowerShell
      - [ ] CLI
      - [ ] Terraform
      - [ ] No refresh required for updates in this PR

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of "no breaking changes"
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist 

> **Applicability**: :warning: 
>
> If your changes encompass only the following scenarios, you should SKIP this section, as these scenarios do not require ARM review.
> - Change to data plane APIs 
> - Adding new properties 
> - All removals

Otherwise your PR may be subject to ARM review requirements. Complete the following:
- [ ] Check this box if any of the following appy to the PR so that the label "ARMReview" and "WaitForARMFeedback" will be added by bot to kick off ARM API Review.  Missing to check this box in the following scenario may result in delays to the ARM manifest review and deployment.
  - Adding a new service
  - Adding new API(s)
  - Adding a new API version
    -[ ] To review changes efficiently, ensure you are using OpenAPIHub to initialize the PR for adding a new version. More details, refer to the [wiki](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/208/OpenAPI-Hub-Adding-new-API-version).

- [ ] Ensure you've reviewed following [guidelines](https://aka.ms/rpguidelines) including [ARM resource provider contract](https://github.com/Azure/azure-resource-manager-rpc) and [REST guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md). Estimated time (4 hours). This is required before you can request review from ARM API Review board.

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If any of the following scenarios apply to the PR, request approval from the Breaking Change Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 
- [ ] Removing API(s) in a stable version
- [ ] Removing properties in a stable version
- [ ] Removing API version(s) in a stable version
- [ ] Updating API in a stable or public preview version with Breaking Change Validation errors
- [ ] Updating API(s) in public preview over 1 year (refer to [Retirement of Previews](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37683/Retirement-of-Previews))

**Action**: to initiate an evaluation of the breaking change, create a new intake using the [template for breaking changes](https://aka.ms/Breakingchangetemplate). Addition details on the process and office hours are on the [Breaking change Wiki](https://dev.azure.com/msazure/AzureWiki/_wiki/wikis/AzureWiki.wiki/37684/Breaking-Changes).

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

fix https://github.com/Azure/azure-rest-api-specs/issues/20500

### CODE REVIEW

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json [hunk 1]
```diff
@@ -112,6 +112,59 @@
           }
         }
       }
+    },
+    "/sip/checkRouting/{targetNumber}": {
```
Comment (prprabhu-ms) [inline specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json:116]: I'd recommend that this be a verb: `/sip:checkRouting` and that `targetNumber` be a required query parameter.

Justification:
* `/sip/checkRouting/{targetNumber}` suggests that the number being checked is a REST resource with CRUD operations. But that is not true. There is no resource backing this query, you can't create / update / delete this object.
* The `checkRouting` verb path infix shows that this is really an operation, not a resource (path parts should be nouns, not verbs).

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json [hunk 2]
```diff
@@ -112,6 +112,59 @@
...
+        ],
+        "summary": "Gets the list of routes matching the target phone number, ordered by priority.",
+        "operationId": "CheckRoutingForNumber",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "targetNumber",
```
Comment (prprabhu-ms) [inline specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json:129]: nit: Is "targetNumber" a domain specific name? I find the adjectives "source" and "destination" more natural in the context of a connection / path etc.

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json [hunk 3]
```diff
@@ -112,6 +112,59 @@
...
+            "required": true,
+            "type": "string",
+            "x-example": "+11234567890"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "query",
+            "name": "$select",
```
Comment (prprabhu-ms) [inline specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json:140]: This should be named `select`, not `$select`: https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#query-options

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json [hunk 4]
```diff
@@ -216,6 +269,24 @@
           }
         }
       }
+    },
+    "RoutingForNumber": {
```
Comment (prprabhu-ms) [inline specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json:273]: naming: Should this be called `RoutesForNumber` (returns a collection with multiple routes)

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json [hunk 5]
```diff
@@ -216,6 +269,24 @@
...
+      "description": "Represents number routing validation details.",
+      "type": "object",
+      "properties": {
+        "targetNumber": {
+          "description": "A string in the E.164 format of a phone number, e.g. +11234567890.",
+          "type": "string"
+        },
+        "matchingRoutes": {
+          "description": "The list of routes whose number patterns are matched by the target number. The routes are displayed and apply in the same order as in SipConfiguration. Only route names are displayed.",
```
Comment (prprabhu-ms) [inline specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json:282]: The discription should not talk about how numbers are displayed in the UI. In particular "Only route names are displayed" is specific to how the API is called and responses used in the UI.

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json [hunk 6]
```diff
@@ -216,6 +269,24 @@
           }
         }
       }
+    },
+    "RoutingForNumber": {
+      "description": "Represents number routing validation details.",
+      "type": "object",
+      "properties": {
+        "targetNumber": {
```
Comment (prprabhu-ms) [inline specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json:277]: Why is this included in the response? the `targetNumber` is required parameter in the API request, so client already has the target number being queried.

Request parameters should only be included in responses in exceptional cases.

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json (hunk 5 above)
Comment (kdruzycki) [reply specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json]: Removed the sentence.

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json (hunk 4 above)
Comment (kdruzycki) [reply specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json]: Changed the name, thanks for the remark

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json (hunk 3 above)
Comment (kdruzycki) [reply specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json]: Fixed

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json (hunk 1 above)
Comment (kdruzycki) [reply specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json]: Used the verb notation and moved the parameter from path to query.

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json (hunk 2 above)
Comment (kdruzycki) [reply specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json]: In the case of PSTN calling I find the expression "target phone number" to be more natural in English than "destination phone number". I'd advocate staying with "target"

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/examples/CheckNumberRouting.json [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+    "targetNumber": "+11234567890",
+    "$select": "name"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "matchingRoutes": [
+          {
+            "name": "Seatle"
```
Comment (DominikMe) [inline specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/examples/CheckNumberRouting.json:14]:
```suggestion
            "name": "Seattle"
```

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json [hunk 8]
```diff
@@ -112,6 +112,59 @@
...
+          "NumberRoutingValidation"
+        ],
+        "summary": "Gets the list of routes matching the target phone number, ordered by priority.",
+        "operationId": "CheckNumberRouting",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
```
Comment (DominikMe) [inline specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json:128]: This doesn't seem to be a path parameter?

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json [hunk 9]
```diff
@@ -112,6 +112,59 @@
...
+            "required": true,
+            "type": "string",
+            "x-example": "+11234567890"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "query",
+            "name": "select",
```
Comment (DominikMe) [inline specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json:140]: Is this feature valuable enough? TrunkRoute seems to have only a few properties and no heavy payloads.

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/examples/CheckRoutingForNumber.json [hunk 10]
```diff
@@ -0,0 +1,24 @@
+{
+  "parameters": {
+    "api-version": "2022-10-01-preview",
+    "endpoint": "https://resourcename.communication.azure.com/",
+    "targetNumber": "+11234567890",
+    "$select": "name"
```
Comment (mbakalov) [inline specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/examples/CheckRoutingForNumber.json]: Should this also be `select` and not `$select`?

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json [hunk 11]
```diff
@@ -112,6 +112,59 @@
           }
         }
       }
+    },
+    "/sip:checkNumberRouting": {
+      "get": {
```
Comment (DominikMe) [inline specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json:117]: The `/sip:checkNumberRouting` syntax suggests that this is a POST action, not a GET, see guideline https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#performing-an-action

Alternatively ,we could model this as a query param filter on a new GET route for TrunkRoutes. Something like GET /sip/routes?matches=+12345556789. But that would mean to introduce `/routes` and `/trunks` subcollection routes in addition to the `/sip` configuration object that contains both.

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json [hunk 12]
```diff
@@ -112,6 +112,59 @@
           }
         }
       }
+    },
+    "/sip:checkNumberRouting": {
```
Comment (mbakalov) [inline specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json:116]: This is formatted as an action, yet is a GET request. [Actions](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#performing-an-action) are always POST though.

Should this maybe be `POST /sip:checkNumberRouting` with payload
```json
{
   "targetPhoneNumber": "12345"
}
```
?

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/examples/CheckNumberRouting.json (hunk 7 above)
Comment (kdruzycki) [reply specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/examples/CheckNumberRouting.json]: fixed

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json (hunk 8 above)
Comment (kdruzycki) [reply specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json]: That's right - it used to be before @prprabhu-ms 's review (namely `/sip/checkRouting/{targetNumber}`) but as Prathmesh said it shouldn't be path since it's not a resource. Just fixed this now

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json (hunk 9 above)
Comment (kdruzycki) [reply specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json:140]: The `select` parameter will enable data to be much more lean - for the UI only the name is necessary.
Let's discuss on today's review

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/examples/CheckRoutingForNumber.json (hunk 10 above)
Comment (kdruzycki) [reply specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/examples/CheckRoutingForNumber.json]: Fixed now

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json (hunk 11 above)
Comment (kdruzycki) [reply specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json:117]: How about `/sip/checkNumberRouting`? I don't want to introduce more subcollections for sip configuration - that would require overhead to support them in the future but here I just want to add the functionality for testing number pattern matching.

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json [hunk 13]
```diff
@@ -216,6 +269,20 @@
...
         }
       }
+    },
+    "RoutesForNumber": {
+      "description": "Represents number routing validation details.",
+      "type": "object",
+      "properties": {
+        "matchingRoutes": {
+          "description": "The list of routes whose number patterns are matched by the target number. The routes are displayed and apply in the same order as in SipConfiguration.",
+          "maxLength": 250,
```
[the remark this answers is not in the document]
Comment (kdruzycki) [reply specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json]: I removed this parameter. It could be `maxItems` but actually there's no need to restrict the number of returned items.

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json (hunk 9 above)
Comment (petrsvihlik) [reply specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json:140]: I wonder what type we should use here. I've run a quick code search and it seems there are occurrences of both - [arrays](https://cs.github.com/Azure/azure-rest-api-specs/blob/a436672b07fb1fe276c203b086b3f0e0d0c4aa24/specification/search/data-plane/Azure.Search/stable/2019-05-06/searchindex.json?q=%22type%22%3A+%22array%22+select#L252) and [comma-separated strings](https://cs.github.com/Azure/azure-rest-api-specs/blob/a436672b07fb1fe276c203b086b3f0e0d0c4aa24/specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/stable/2018-05-05/AlertsManagement.json#L857).

However, according to the [guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#query-options) the type should be a string array.

```json
"name": "$select",
"in": "query",
"type": "array",
"items": {
  "type": "string"
},
```

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json [hunk 14]
```diff
@@ -112,6 +112,59 @@
...
+          "200": {
+            "description": "A list of matching routes",
+            "schema": {
+              "$ref": "#/definitions/RoutesForNumber"
+            }
+          },
+          "default": {
+            "description": "Failure",
+            "schema": {
+              "$ref": "../../../Common/stable/2021-03-07/common.json#/definitions/CommunicationErrorResponse"
```
Comment (petrsvihlik) [inline specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json:157]: It would be great if you could update the version of Common to the latest stable - https://github.com/Azure/azure-rest-api-specs/tree/main/specification/communication/data-plane/Common/stable/2022-07-13

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json (hunk 11 above)
Comment (petrsvihlik) [reply specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json:117]: How about `GET /sip/routesByPriority`?

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json (hunk 9 above)
Comment (kdruzycki) [reply specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json:140]: @petrsvihlik I changed the type to array. Since it's formatted as a csv inside the query, it comes down to a comma-separated string, in practice

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json (hunk 14 above)
Comment (petrsvihlik) [reply specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json]: Thanks, it'd be good to update all the references in the file.

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json (hunk 9 above)
Comment (kdruzycki) [reply specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json]: Select removed as discussed during internal review meeting

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json (hunk 14 above)
Comment (kdruzycki) [reply specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json]: Updated in the newer commits

File: specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json (hunk 12 above)
Comment (kdruzycki) [reply specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json]: @DominikMe @mbakalov @petrsvihlik @prprabhu-ms 
Based on the internal review we decided to make it a POST action `POST /sip:matchRoutesForNumber`
One more question from me is maybe we can keep `targetPhoneNumber` as a `query` parameter... It fits the [guidelines](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#performing-an-action) and I think is more abstract than specifying JSON body payload, and `targetPhoneNumber` is not data that is submitted to be stored/make any permanent change in the resource - so maybe no need to hide it in `body`. What do you think?

> :ballot_box_with_check: **YOU SHOULD** pattern your URL like this to perform an action on a collection
> **URL Pattern**
> ```text
> https://.../<resource-collection>:<action>?<input parameters>
> ```
> 
> **Example**
> ```text
> https://.../users:grant?access=read
> ```

Comment (kdruzycki) [reply specification/communication/data-plane/SipRouting/preview/2021-05-01-preview/communicationservicessiprouting.json]: Alright, following what we agreed on in the review chat, will keep `targetPhoneNumber as `query` remembering to support bot `%2B` and `+` in the URI.

**Comment by kdruzycki** [comment]
Hello @mikekistler @markweitzel! I made the changes suggested at our review meeting. The code is now ready for final review and merge. More info in the issue comment: https://github.com/Azure/azure-rest-api-specs/issues/20500#issuecomment-1261718724

**Review by mikekistler — APPROVED** [review]
Looks good! 👍

**Review by weidongxu-microsoft — APPROVED** [review]

**Comment by mikekistler** [comment]
@jhendrixMSFT This one is good to merge.

File: specification/communication/data-plane/SipRouting/preview/2022-10-01-preview/communicationservicessiprouting.json [hunk 15]
```diff
@@ -0,0 +1,369 @@
...
+            "$ref": "./examples/GetSipConfiguration.json"
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "SipConfiguration"
+        ],
+        "summary": "Patches SIP configuration for resource.",
+        "operationId": "SipRouting_Patch",
```
Comment (jhendrixMSFT) [inline specification/communication/data-plane/SipRouting/preview/2022-10-01-preview/communicationservicessiprouting.json:58]: We typically use the verb `Update` instead of patch along with types that have the `Properties` suffix.

File: specification/communication/data-plane/SipRouting/preview/2022-10-01-preview/communicationservicessiprouting.json [hunk 16]
```diff
@@ -0,0 +1,369 @@
...
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "in": "body",
+            "name": "body",
+            "description": "Sip configuration patch object.",
+            "schema": {
+              "$ref": "#/definitions/SipConfigurationPatch"
```
Comment (jhendrixMSFT) [inline specification/communication/data-plane/SipRouting/preview/2022-10-01-preview/communicationservicessiprouting.json:74]: So this, and any containing types, would change to `SipConfigurationProperties`.

File: specification/communication/data-plane/SipRouting/preview/2022-10-01-preview/communicationservicessiprouting.json (hunk 15 above)
Comment (kdruzycki) [reply specification/communication/data-plane/SipRouting/preview/2022-10-01-preview/communicationservicessiprouting.json:58]: This part hasn't been changed. It's the same as in previous version of the API that's already in the main branch.
Look at this line in the main branch
https://github.com/Azure/azure-rest-api-specs/blob/ec278eb936001b993e0413d66d8cc88e73540331/specification/communication/data-plane/SipRouting/preview/2022-09-01-preview/communicationservicessiprouting.json#L58
It is beyond of the scope of the issue #20500 to change other endpoints not related to the `/sip:testRoutesWithNumber` endpoint
